hat/assets/js/apps/Iaso/domains/teams/components/CreateEditTeam.tsx

```diff
@@ -89,13 +89,6 @@ export const CreateEditTeam: FunctionComponent<Props> = ({
         useGetProjectsDropDown();
     const { data: profliesDropdown, isFetching: isFetchingProfiles } =
         useGetProfilesDropdown();
-    const { data: teamsDropdown = [], isFetching: isFetchingTeams } =
-        useGetTeamsDropdown(
-            {
-                project,
-            },
-            id,
-        );
```

Comment:
This is still used to compute `availableChildren`

Reply:
Fixed.